### DISCUSSION TITLE
Checkbox in Formik

### DISCUSSION THREAD

**Discussion by yuchuga** [jaredpalmer/formik#3605] [Q&A]
Hi, Formik output checkbox value as true (if checked) or false (unchecked). How can I convert the output to '1' or '0' for checkbox as database query require it to be string format instead of boolean. I am using Yup schema to define it as follows:

    terms: Yup.boolean()
    .required('Required')
    .oneOf([true], "Pls accept terms and conditions"),

[bombillazo](https://github.com/bombillazo) pls help. Thanks!

**Comment by quick007** [comment]
You could 
1. do nothing, as you're already validating the terms and conditions client side and there's not exactly a reason to save it to the database (as all users will have to accept it anyways)
2. do something like this
```js
<Formik onSubmit((data) => writeToDb(uID, {acceptedTerms: (data.terms ? "1" : "0"), /*...*/ }
//...
